shared/mail/mail.go

```diff
@@ -307,6 +316,14 @@ func sendMail(c smtpClient, mail mailData, date time.Time, config *SMTPConfig) e
 		"Precedence":                {"bulk"},
 	}
 
+	if len(mail.category) > 0 {
```

Comment:
Let's change this to `mail.category != ""`. Refer to: https://developers.mattermost.com/contribute/more-info/server/style-guide/#empty-string-check